### PR TITLE
Email confirmation

### PR DESCRIPTION
**Pull request by alex-tanasov**

### CODE REVIEW

File: src/api/user/auth.js [hunk 1]
```diff
@@ -1,30 +1,52 @@
...
+  address: Joi.string(),
+  state: Joi.string()
+})
+exports.loginSchema = Joi.object().keys({
   email: Joi.string().email().required(),
   password: Joi.string().min(8).required()
 })
 
-exports.register = async (req, res, next) => {
+exports.register = wrapper(async (req, res, next) => {
```
Comment (gi4c0) [inline src/api/user/auth.js]: You don't need try catch if you use wrapper and vice versa

File: src/api/user/auth.js [hunk 2]
```diff
@@ -34,6 +56,18 @@ exports.login = wrapper(async (req, res, next) => {
...
 
-  const token = await jwt.sign({ email: user.email }, secret)
+  const token = await jwt.sign({ email: user.email }, secret, { expiresIn: '1d' })
+
+  if (!user.isVerified) return next({ httpCode: 401, message: 'Please confirm your email' })
   res.json({ token: token })
 })
+
+exports.confirm = wrapper(async (req, res, next) => {
+  await jwtPromiseVer(req.query.token, 'registration').then(async (result) => {
```
Comment (gi4c0) [inline src/api/user/auth.js]: use only async / await

Comment (alex-tanasov) [reply src/api/user/auth.js]: you mean - only one?

File: src/utils/mail.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+  from: 'Alex_T <o.tanasov@do-it.co>', // sender address
+  to: 'e3365940@nwytg.net', // list of receivers
+  subject: 'Confirm your email' // Subject line
+}
+
+// send mail
+exports.sendMail = (token, email) => {
+  const html = `<b>Hi ${email}</b><br><a href = http://localhost:3000/api/users/confirm-token?token=${token}>Ссыыыылка`
+  transporter.sendMail({ ...mailOptions, html: html })
+    .then(console.log)
```
Comment (gi4c0) [inline src/utils/mail.js]: unnecessary `console.log` in then

**Pull request merged**